wagtail/wagtaildocs/wagtail_hooks.py

```diff
@@ -59,3 +61,17 @@ def register_permissions():
 @hooks.register('register_rich_text_link_handler')
 def register_document_link_handler():
     return ('document', DocumentLinkHandler)
+
+
+class DocumentsSummaryItem(SummaryItem):
+    order = 300
+    template = 'wagtaildocs/homepage/site_summary_documents.html'
+
+    def get_context(self):
+        return {
+            'total_docs': Document.objects.count(),
+        }
+
```

Comment:
Isn't it better to keep it visually grouped with the hook definition, to indicate that they're dealing with the same thing?